configs/env-example

```diff
@@ -1,3 +1,6 @@
+# Set time zone for the same expected from Openshift.
+TZ=UTC
+
 # Project Specific Env
```

Comment:
Can we add BYPASS_CRA_INCOME_VERIFICATION=false in the sample env file(may be with a comment) which will be helpful for any new person setting up the application locally.

Reply:
Comments added.